chromatic/rainbows/actions/remove_trends.py

```diff
@@ -0,0 +1,120 @@
+from ...imports import *
+
+
+def remove_trends(
+    self, method="median_filter", **kw
+):  # model=None,win_length=None,polyorder=None,butter_cutoff=None,butter_fs=None, butter_order=9,
+    """
+    A quick tool to approximately remove astrophysical signals,
+    usually by some combination of smoothing/filtering.
+
+    Parameters
+    ----------
+    method : str
+        What method should be used to make an approximate model
+        for smooth trends that will then be subtracted off?
+
+        `differences` will do an extremely rough filtering
+        of replacing the fluxes with their first differences.
+        Trends that are smooth relative to the noise will
+        be removed this way, but sharp features will remain.
+        Required keywords:
+            None.
+
+        `median_filter` is a wrapper for scipy.signal.median_filter.
+        It smoothes each data point to the median of its surrounding
+        points in time and/or wavelength. Required keywords:
+            `size` = centered on each point, what shape rectangle
+            should be used to select surrounding points for median?
+            The dimensions are (nwavelengths, ntimes), so `size=(3,7)`
+            means we'll take the median across three wavelengths and
+            seven times. Default is `(1,5)`.
+
+        `savgol_filter` is a wrapper for scipy.signal.savgol_filter.
+        It applies a Savitzky-Golay filter for polynomial smoothing.
+        Required keywords:
+            `window_length` = the length of the filter window,
+            which must be a positive odd integer. Default is `5`.
+            `polyorder` = the order of the polynomial to use.
+            Default is `2`.
+
+        `custom` allow users to pass any fluxlike array of model
+        values for an astrophysical signal to remove it. Required
+        keywords:
+            `model` = the (nwavelengths, ntimes) model array
+
+    kw : dict
```

Comment:
@Pat-Wachiraphan , I changed the inputs to try to make them a little less confusing and flexible. Now, extra keywords can be supplied, and we just pick out what we need from those dictionaries if necessary. It's similar to your solution of simply defining all the keywords you might possibly need, but I think it just makes things feel a little tidier.